sandbox/package-lock.json

```diff
@@ -25,364 +25,278 @@
       }
     },
     "@hapi/accept": {
```

Comment:
Hope this is auto generated file.

Reply:
It is -- generated when we do a 
$ npm install
or
$ make install
And we keep it in vcs so that dependencies are the same across different machines

Reply:
It is yup 🙂, the package-lock.json file is managed by npm utilising package.json.